src/main/java/org/apache/mesos/scheduler/plan/BlockFactory.java

```diff
@@ -11,6 +12,6 @@
  * detected for a Task.
```

Comment:
Will this be provided all defined/available TaskSpecifications at once, or just some subset? Could specify this in the javadoc for reference.